code/modules/admin/verbs/adminhelp_vr.dm

```diff
@@ -36,3 +38,37 @@
 	usr.verbs -= /client/verb/adminspice
 	spawn(6000)
 		usr.verbs += /client/verb/adminspice	// 10 minute cool-down for spice request
+*/
+
+/mob/verb/Spice(msg as text) //CHOMPStation Addition 14/1/20 Ksc - The improved request spice command.
```

Comment:
Also would have this be like Chomp Addition Start/End for the whole snippet but since it's the end of the file, it'll be fine for now. :>